python/private/bzlmod/pip.bzl

```diff
@@ -31,6 +31,11 @@ load("//python/private:parse_whl_name.bzl", "parse_whl_name")
 load("//python/private:version_label.bzl", "version_label")
 load(":pip_repository.bzl", "pip_repository")
 
+def _major_minor_version(version):
+    version = full_version(version)
```

Comment:
Is the full_version() call just kept for the error checking? The logic it incurs is basically:
```
version = "3.10"
full_version = mapping[version] # "3.10.1" or error if unrecognized version
major_minor, _, _ = fulL_version.rpartion(full_version) # back to 3.10
```

If so, I'm fine with that for now. Lets just drop a comment about that.

A version-parsing function would be better TBH. I had to do this same "parse the dots" game in the PR to generate the config settings. But that can go in a separate PR.

Reply:
Yeah, just wanted to get something functional before I spend cleaning up the API.

Reply:
Changed the function to do actual version parsing and if this is something that looks OK, we can move it to `python/private/semver.bzl#parse_semver` or something similar.